sherpa/data.py

```diff
@@ -1080,7 +1124,9 @@ def indep(self, val):
         self._data_space = self._init_data_space(self._data_space.filter, *val)
         self._clear_filter()
 
-    def get_indep(self, filter=False):
+    def get_indep(self,
+                  filter: bool = False
+                  ) -> Optional[Union[tuple[np.ndarray], tuple[None]]]:
```

Comment:
Hmmm, can it be a `None` or is it only `tuple[None]`, in which case the `Optional` is not needed.

Reply:
This was from an earlier version. The code now better types the optional behaviour (still not perfect, but better than it was).